### ISSUE TITLE
ChatInterface is broken on 1.3.7rc2

### ISSUE THREAD

**Issue by pmeier** [holoviz/panel#6241]
When you try to send a message you are instantly hit with an error.

```python
import panel as pn
from panel.chat import ChatInterface

pn.extension()


def echo(contents, user, instance):
    return f"{user} said: '{contents}'"


chat_interface = ChatInterface(callback=echo, callback_exception="verbose").show()
```

```
Traceback (most recent call last):
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/chat/feed.py", line 486, in _prepare_response
    self._callback_state = CallbackState.RUNNING
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/contextlib.py", line 126, in __exit__
    next(self.gen)
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/param/parameterized.py", line 307, in batch_call_watchers
    parameterized.param._batch_call_watchers()
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/param/parameterized.py", line 2506, in _batch_call_watchers
    self_._execute_watcher(watcher, events)
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/param/parameterized.py", line 2468, in _execute_watcher
    watcher.fn(*args, **kwargs)
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/reactive.py", line 374, in _param_change
    self._apply_update(named_events, properties, model, ref)
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/reactive.py", line 302, in _apply_update
    self._update_model(events, msg, root, model, doc, comm)
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/layout/base.py", line 119, in _update_model
    super()._update_model(events, msg, root, model, doc, comm)
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/reactive.py", line 623, in _update_model
    super()._update_model(events, msg, root, model, doc, comm)
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/reactive.py", line 318, in _update_model
    model_val = getattr(model, attr)
  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/bokeh/core/has_props.py", line 357, in __getattr__
    return super().__getattribute__(name)
AttributeError: 'Card' object has no attribute '_callback_state'
```

**Comment by philippjfr** [comment]
Thanks, this seems to have been an issue with the backporting. Will fix now.

**Comment by philippjfr** [comment]
@ahuang11 Any idea why this isn't being caught by the test suite?

**Comment by philippjfr** [comment]
Fixed in 1.3.7rc3.

**Comment by ahuang11** [comment]
> @ahuang11 Any idea why this isn't being caught by the test suite?

I realized that the errors were piped to the chat logs, rather than raising: https://github.com/holoviz/panel/pull/6245
